src/phoenix/server/api/helpers/playground_spans.py

```diff
@@ -224,12 +224,49 @@ def _input_value_and_mime_type(input: Any) -> Iterator[tuple[str, Any]]:
     yield INPUT_VALUE, safe_json_dumps(input_data)
 
 
-def _output_value_and_mime_type(output: Any) -> Iterator[tuple[str, Any]]:
-    yield OUTPUT_MIME_TYPE, JSON
-    yield OUTPUT_VALUE, safe_json_dumps(jsonify(output))
+def _merge_tool_call_chunks(
+    chunks_by_id: defaultdict[str, list[ToolCallChunk]],
+) -> Union[list[dict[str, Any]], dict[str, Any]]:
+    merged_tool_calls = []
+
+    for tool_id, chunks in chunks_by_id.items():
+        # Get the name from the first chunk since it should be the same for all chunks
+        first_chunk = chunks[0]
```

Comment:
```suggestion
        function_name = chunks[0].function.name
```

Closer to comment, less likely for comment to get orphaned.

Reply:
Might be worth a check for empty chunks